packages/internals/helpers/build.ts

```diff
@@ -1,6 +1,30 @@
+import type * as esbuild from 'esbuild'
+import fs from 'fs'
+import path from 'path'
+
 import { build } from '../../../helpers/compile/build'
 import { esmSplitCodeToCjs } from '../../../helpers/compile/plugins/esmSplitCodeToCjs'
 
+const copySchemaEngineWasmPlugin: esbuild.Plugin = {
```

Comment:
❓ I am not sure if this should be done like this. But otherwise the tests had trouble to find the WASM files.

@aqrln can you have look into this approach briefly?

Reply:
Yeah this looks about right given that we bundle `packages/internals`. A better approach though would be to stop bundling `packages/internals` IMO, I don't see how it's useful or why we do it, while there clearly are downsides, like this one, or the fact it can lead to code bloat and duplication of shared dependencies when the bundle is being re-bundled again as part of the CLI or client. Only the top level user-facing packages should be bundled.

Reply:
Ok - then I'll leave this for now and we optimize later. :+1: